### ISSUE TITLE
Type annotations of normalize() are incorrect

### ISSUE THREAD

**Issue by NicolasHug** [pytorch/vision#5046]
I'm seeing the following typing error:

```
torchvision/prototype/transforms/_presets.py:114: error: Argument "mean" to
"normalize" has incompatible type "float"; expected "List[float]"  [arg-type]
            img1 = F.normalize(img1, mean=0.5, std=0.5)
```

This is incorrect, passing single numbers is explicitly supported by `noramlize()`

https://github.com/pytorch/vision/blob/47bd962069ba03f753e7ba711cb825317be0b00a/torchvision/transforms/functional.py#L340-L345

**Comment by datumbox** [comment]
Probably a limitation from the days that JIT didn't support Union? Worth checking if a `Union[float, List[float]]` fixes the situation.
